tools/update_extern.py

```diff
@@ -0,0 +1,169 @@
+#!/bin/env python
+
+"""
+This script is used to update the files in `sunpy/extern` while making new releases.
+"""
+
+import os
+import shutil
+from zipfile import ZipFile
+import requests
+from pathlib import Path
+
+# get parent directory of sunpy
+SUNPY_DIR = Path(__file__).parent.parent
+
+
+# List of packages to update
+PACKAGES = ["appdirs", "distro", "inflect", "parse"]
+
+# Dictionary to store the authors of the packages
+AUTHORS = {
+    PACKAGES[0]: "ActiveState",
+    PACKAGES[1]: "python-distro",
+    PACKAGES[2]: "jaraco",
+    PACKAGES[3]: "r1chardj0n3s",
```

Comment:
Having the keys as part of the `PACKAGES` list still means two updates if adding another package as you would have to update the `PACKAGES` list and add a `PACKAGES[4]` key to the `AUTHORS` dictionary. What I'm thinking is that, since we have single file packages maybe we could have a dictionary such as:

```py
PACKAGES = {
    "inflect": ["jaraco", "inflect/__init__.py"],
    ...
}
```
This tells us the package name, the author, and the file within the package that we want to copy to `sunpy/extern/<package name>.py`. If that works, it should mean that we could remove the package specific code from the script (i.e. remove `update_extern` and move that logic to `move_file`), and if we ever want to vendor a new package into extern it should be as simple as adding it to the `PACKAGES` dictionary and running this script.

Reply:
I hadn't read this comment before the push, but I tried to use a single dictionary.

Reply:
I am with Conor on this, that we we store the information we need in the dict, the author, the files we need to copy over and it should make the script simpler.